cmake/main.cmake

```diff
@@ -111,28 +111,28 @@ endif()
 # from seastar/CMakeLists.txt. unfortunately this snippet doesn't appear to be
 # installed along with the rest of seastar.
 function (seastar_generate_swagger)
-  set (one_value_args TARGET VAR IN_FILE OUT_FILE)
+  set (one_value_args TARGET VAR IN_FILE OUT_DIR)
   cmake_parse_arguments (args "" "${one_value_args}" "" ${ARGN})
-  get_filename_component (out_dir ${args_OUT_FILE} DIRECTORY)
-  if(REDPANDA_DEPS_ONLY)
-    set (generator "${REDPANDA_DEPS_INSTALL_DIR}/bin/seastar-json2code.py")
-  else()
-    find_program(GENERATOR "seastar-json2code.py")
-    set (generator "${GENERATOR}")
-  endif()
+  get_filename_component (in_file_name ${args_IN_FILE} NAME)
+  find_program(GENERATOR "seastar-json2code.py")
+  set (generator "${GENERATOR}")
+  set (header_out ${args_OUT_DIR}/${in_file_name}.hh)
```

Comment:
is the .hh intentional? seems like a lot of code churn - or is that the convention for all "generated" code. meaning our rpcgen, etc would also add the .hh

Reply:
For posterity: https://github.com/scylladb/seastar/blob/c215b152a32cc10c3dd88fd77814cc1bdfa3e185/CMakeLists.txt#L430

Reply:
ah lgtm. thanks for sharing.